src/main/resources/db/migration/V08__arrangor-organisasjonsnummer.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE arrangor ADD COLUMN organisasjonsnummer VARCHAR NOT NULL DEFAULT 'UKJENT_ORGANISASJONSNUMMER';
```

Comment:
Bruker en midlertidig default her isteden for null, siden da slipper jeg å endre noe i koden, og den skal helst være `not null`.